src/main/java/magicbees/main/Config.java

```diff
@@ -146,6 +149,8 @@ public class Config {
     public static ItemMagicHiveFrame hiveFrameNecrotic;
     public static ItemMagicHiveFrame hiveFrameTemporal;
     public static ItemMagicHiveFrame hiveFrameOblivion;
+    public static ItemBloodFrame hiveFrameBlood;
```

Comment:
You should probably store them as `Item` to not load the class if there is no dependency, not sure tho as im on the phone

Reply:
Nope didn't fix it, any other ideas?